docs/root/intro/version_history.rst

```diff
@@ -26,6 +26,8 @@ Version history
 * rate-limit: added :ref:`configuration <envoy_api_field_config.filter.http.rate_limit.v2.RateLimit.rate_limited_as_resource_exhausted>`
   to specify whether the `GrpcStatus` status returned should be `RESOURCE_EXHAUSTED` or
   `UNAVAILABLE` when a gRPC call is rate limited.
+* rate-limit: removed support for the legacy ratelimit service and made the data-plane-api
+  proto (:repo:`api/envoy/service/ratelimit/v2/rls.proto`) based implementation default.
```

Comment:
Any interest in actually just getting rls.proto running through the doc generator so that we can ref link directly to the root?  Then we could update https://www.envoyproxy.io/docs/envoy/latest/configuration/rate_limit#config-rate-limit-service also?

Reply:
Addressed in c79ef8e.